language/move-prover/src/boogie_helpers.rs

```diff
@@ -122,8 +122,25 @@ pub fn boogie_local_type(ty: &Type) -> String {
     }
 }
 
-/// Create boogie type check boolean expression.
-pub fn boogie_type_check_expr(env: &GlobalEnv, name: &str, ty: &Type) -> String {
+/// A value indicating how to perform type checks.
```

Comment:
Is this comment misleading?  I thought this enum value indicates what kind of well-formedness assumption (or perhaps assertion also?) to generate.

Reply:
Fixed once I push my local commit.